articles/active-directory/develop/scenario-desktop-acquire-token-wam.md

```diff
@@ -17,139 +17,135 @@ ms.custom: aaddev, devx-track-python
 
 ## WAM value proposition
 
 Using an authentication broker such as WAM has numerous benefits.
 
-- Enhanced security (your app doesn't have to manage the powerful refresh token)
+- Enhanced security. See [token protection](https://learn.microsoft.com/azure/active-directory/conditional-access/concept-token-protection)
 - Better support for Windows Hello, Conditional Access and FIDO keys
 - Integration with Windows' "Email and Accounts" view
-- Better Single Sign-On (users don't have to reenter passwords)
+- Better Single Sign-On 
+- Ability to sign-in silently with the current Windows account
 - Most bug fixes and enhancements will be shipped with Windows
 
 ## WAM limitations
 
+- Available on Win10+ and Win Server 2019+. On Mac, Linux, and earlier versions of Windows, MSAL will automatically fall back to a browser.
 - B2C and ADFS authorities aren't supported. MSAL will fall back to a browser.
-- Available on Win10+ and Win Server 2019+. On Mac, Linux, and earlier versions of Windows, MSAL will fall back to a browser.
-- Not available on Xbox.
 
 ## WAM calling pattern
 
 You can use the following pattern to use WAM.
 
 ```csharp
-// 1. Configuration - read below about redirect URI
-var pca = PublicClientApplicationBuilder.Create("client_id")
-              .WithBroker()
-              .Build();
-
-// Add a token cache, see https://learn.microsoft.com/azure/active-directory/develop/msal-net-token-cache-serialization?tabs=desktop
-
-// 2. GetAccounts
-var accounts = await pca.GetAccountsAsync();
-var accountToLogin = // choose an account, or null, or use PublicClientApplication.OperatingSystemAccount for the default OS account
-
-try
-{
-    // 3. AcquireTokenSilent 
-    var authResult = await pca.AcquireTokenSilent(new[] { "User.Read" }, accountToLogin)
-                              .ExecuteAsync();
-}
-catch (MsalUiRequiredException) // no change in the pattern
-{
-    // 4. Specific: Switch to the UI thread for next call . Not required for console apps.
-    await SwitchToUiThreadAsync(); // not actual code, this is different on each platform / tech
-
-    // 5. AcquireTokenInteractive
-    var authResult = await pca.AcquireTokenInteractive(new[] { "User.Read" })
-                              .WithAccount(accountToLogin)  // this already exists in MSAL, but it is more important for WAM
-                              .WithParentActivityOrWindow(myWindowHandle) // to be able to parent WAM's windows to your app (optional, but highly recommended; not needed on UWP)
-                              .ExecuteAsync();
-}
+    // 1. Configuration - read below about redirect URI
+    var pca = PublicClientApplicationBuilder.Create("client_id")
+                    .WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows))
```

Comment:
Where do we explain how to use the broker options?

Reply:
We don't. We hope it's self explanatory. All the options there are advanced scenarions.